src/react/workflow/ReplicationForm.tsx

```diff
@@ -421,6 +421,11 @@ const RenderDestination = ({
       forceLabelWidth={forceLabelWidth}
     >
       <Controller
+        // This required hack is there to indicates to FormSection to remove the
+        // (optional) text.
+        // eslint-disable-next-line
+        // @ts-ignore
+        required
```

Comment:
I'm a bit surprised that this hack would works 🤔.

Reply:
And yet it *is* working ha ha 😁

Reply:
The idea is to provide a `required` prop into a child (any child in fact) because `FormSection` has an algorithm which is roughly:

```typescript
const children = Children.toArray(props.children)
const required = !!children.find(c => 'required' in c.props)
```

Even if the child does not use `required`, it *is there* and thus, `FormSection` will not display `(optional)`. Why wouldn't it work in your mind?

Reply:
I thought we were relying on a context defined on the input! So alright !